src/cmd.ts

```diff
@@ -319,13 +320,35 @@ async function cmdBuild(argv: BuildCmdArgvType) {
     });
   };
 
+  const buildRSSFeed = () => {
+    const rssPath = path.join(outputPath, "rss.xml");
+    const feed = new RSS({
+      title: appContext.title,
+      description: appContext.title,
+      feed_url: rssPath,
+      site_url: outputPath,
+      pubDate: new Date(),
```

Comment:
Is this date the date the RSS feed was created/updated? Would this ever need to be configurable or set to something more static (e.g., some previously published date)?

Reply:
I think it's fine to have the feed pubDate here be the latest time that you run the build command.